readme.md

```diff
@@ -177,6 +177,7 @@
 - [ActionScript 3](https://github.com/robinrodricks/awesome-actionscript3#readme) - Object-oriented language targeting Adobe AIR.
 - [Eta](https://github.com/sfischer13/awesome-eta#readme) - Functional programming language for the JVM.
 - [Idris](https://github.com/joaomilho/awesome-idris#readme) - General purpose pure functional programming language with dependent types influenced by Haskell and ML.
+- [Ada/SPARK](https://github.com/ohenley/awesome-ada#readme) - Ada/SPARK is a modern programming language designed for large, long-lived applications where reliability and efficiency are essential. 
```

Comment:
Seeing as the list is called Awesome Ada, I don't think the title `Ada/SPARK` is correct here.

Reply:
I do not get it. 

Python, Javascript, Swift, Rust, Go, Idris (do not have time to check the countless others) ... they are all called 'awesome-XYZ' and listed only as their plain name XYZ? 

eg -> [Idris](https://github.com/joaomilho/awesome-idris#readme)

Ada/SPARK is a programming language and it is its name.

Reply:
I'm not talking about the `awesome-` prefix. I'm just confused what the actual name is. The name written here is `Ada/SPARK`. The PR title is just `Ada`. The repo name is `awesome-ada`, not `awesome-ada-spark`. In your list it's written as both `Ada`, `Ada/SPARK`, `SPARK/Ada`. So which is it? You should be consistent about the naming everywhere.

Reply:
Ah ok. SPARK is a recent 'subset' of Ada but of great importance. That is why usually people refer to one or the other, like I did. The core stays Ada but SPARK is a major addition for its formal verification and it is really tied to the latest Ada standard (2012). It is a bit of particular situation if you want my opinion.

Anyway, I renamed the PR to match the description and my list description refers to Ada/SPARK too. If I am not mistaken, things should are consistent now.

Thank you for your time and concern.